Sources/Main.h

```diff
@@ -218,9 +218,11 @@ class TX584Form : public TForm
     void __fastcall ResetItemClick(TObject *Sender);
     void __fastcall HelpItemClick(TObject *Sender);
     void __fastcall AboutItemClick(TObject *Sender);
-
 private:	// User declarations
     void GetSelection(int &SelStart, int &SelEnd);
+
+    TListItem *LastTopItem;
```

Comment:
Wouldn't it be better to simply use `TPoint LastTopItemPosition` instead of `LastTopItem` and `LastTopItemLeft`? Coordinates of the upper left corner of the list item can be obtained via `Position` property.

Reply:
Top item changes when user scrolls `CodeListView`

Reply:
I tried to compare `LastTopItem->Position` with saved `LastTopItemPosition` and it also worked properly, but if I won't save `LastTopItem` and will compare with `CodeListView->TopItem->Position` I would not stop comment editing since `TopItem` changes when user scrolls `CodeListView` vertically

Reply:
Ok, let's merge this hack as it is :smile: